### DISCUSSION TITLE
I can't change iconify flat-color-icon

### DISCUSSION THREAD

**Discussion by IlliaPikq** [arthelokyo/astrowind#443] [Q&A]
I'm encountering an error after replacing the icon on the Features widget with one from the flat-color-icon set obtained from their website. The error message states:

```The 'flat-color-icons' set is not configured to include an icon named 'negative-dynamic'. Do you need to add it to your configuration?```

Do I need to add this icon locally, or is there another way to enable it? I really want to use colored icons 😣😣

**Comment by prototypa** [comment]
Hi @IlliaPikq 

This is because for the `flat-color-icons` icon library, **AstroWind** only define some icons in `astro.config.js`. You can include the entire library or add those icons.

Code in: https://github.com/onwidget/astrowind/pull/442

For Example, for add full library:

```json
{
  "flat-color-icons": ["*"]
}
```

**Reply by IlliaPikq** [reply]
Oh, so that's how it works. Thank you for your fast response
